osu.Game.Tests/NonVisual/ClosestBeatDivisorTest.cs

```diff
@@ -17,7 +17,7 @@ public class ClosestBeatDivisorTest
         public void TestExactDivisors()
         {
             var cpi = new ControlPointInfo();
-            cpi.Add(-1000, new TimingControlPoint { BeatLength = 1000 });
+            cpi.Add(0, new TimingControlPoint { BeatLength = 1000 });
```

Comment:
Likewise, can you explain why the changes in this file were required?

Reply:
Test was working from point's start time so created objects on negative time. It can't work now (will round to 0 and upper) so i didn't spend too much time investigating this, just moved all operations to positive time.